test/spmd/test_xla_distributed_checkpoint.py

```diff
@@ -66,18 +67,22 @@ def _save_and_restore(self,
     tmpdir = tempfile.mkdtemp()
 
     # Save an unsharded model using the provided save planner
+    model_in_state_dict = model_in.state_dict(
+    ) if not is_sharded_cpu_state_dict else _sharded_cpu_state_dict(
+        model_in.state_dict())
```

Comment:
This line is a little hard to follow due to the formatting, could we break it into multiple lines:

```
model_in_state_dict = model_in.state_dict()
if is_sharded_cpu_state_dict:
  model_in_state_dict = _sharded_cpu_state_dict(model_in_state_dict)
```